src/Console/Installer.php

```diff
@@ -179,12 +179,14 @@ public static function setSecuritySalt($dir, $io)
 
         if ($count == 0) {
             $io->write('No Security.salt placeholder to replace.');
```

Comment:
I don't think this improves readability, as mentioned [here](https://github.com/cakephp/cakephp/pull/9114#issuecomment-232915971)